### ISSUE TITLE
[Java] parquet-arrow is broken due to different JDK version

### ISSUE THREAD

**Issue by asfimport** [apache/parquet-java#2158]
Parquet was recently update to use Arrow 0.8.0 to resolve this JIRA: https://issues.apache.org/jira/browse/PARQUET-1128 but unfortunately the issue is actually not resolved because Arrow 0.8.0 uses JDK 7 and Parquet uses JDK 8.

I have filed a Jira against Arrow to upgrade to JDK 8: https://issues.apache.org/jira/browse/ARROW-2498

Once this is done, we will need to update parquet-arrow to use the new version of arrow.

**Reporter**: [Andy Grove](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=andygrove) / @andygrove
#### Related issues:
- [[Java] Upgrade to Arrow 0.10.0](https://issues.apache.org/jira/browse/PARQUET-1390) (duplicates)

<sub>**Note**: *This issue was originally created as [PARQUET-1278](https://issues.apache.org/jira/browse/PARQUET-1278). Please see the [migration documentation](https://issues.apache.org/jira/browse/PARQUET-2502) for further details.*</sub>

**Comment by asfimport** [comment]
[Nándor Kollár](https://issues.apache.org/jira/browse/PARQUET-1278?#comment-16740441) / @nandorKollar:
@andygrove  I resolved this Jira since it looked like it is a duplicate of PARQUET-1390

Feel free to reopen if I was wrong.
